data/assets/scene/solarsystem/heliosphere/bastille_day/fluxnodeslegend.asset

```diff
@@ -9,21 +9,44 @@ local TexturesPath = asset.syncedResource({
 })
 
 local legend = {
-  Identifier = "Legendfluxnodes",
+  Identifier = "LegendFluxnodes",
```

Comment:
Very nitpicky, but above you treaded flux nodes as two words (which I think Is correct?). So this should be `LegendFluxNodes`?